pkg/dbutils/postgresql_utils.go

```diff
@@ -215,3 +217,7 @@ SELECT ID FROM %s WHERE namespace=$1 AND name=$2
 func (m *PostgresAssetManager) GetDB() PostgresDB {
 	return m.DB
 }
+
+func (m *PostgresAssetManager) GetKubeappsNamespace() string {
+	return m.kubeappsNamespace
```

Comment:
In theory nor the `GetDB` or this `GetKubeappsNamespace` are needed if we expose those properties but I am fine anyway. Is there a reason why you added this apart than for standardization?

Reply:
Yeah, because when the initial postgres integration was done, the `PostgresAssetManagerIface` interface was defined above and embedded in the `postgresAssetManager` struct at https://github.com/kubeapps/kubeapps/blob/master/cmd/assetsvc/postgresql_utils.go#L34 . Since you can't add properties to an interface, only methods, this means that when handling an instance of `postgresAssetManager`, you can only get to the methods defined on the interface, not the properties of the underlying struct implementing that interface, without adding type conversions etc.

So yes, I do think we could clean these up and remove a layer or two - there seem to be multiple private implementations of both postgres and mongo (for assetsvc and asset-sync) which I don't think are necessary. But I've not attempted to do that work here.